src/metabase/driver/generic_sql/query_processor.clj

```diff
@@ -168,17 +168,40 @@
         form
         (recur form more)))))
 
+(def histogram-type? #{:type/Latitude :type/Longitude})
```

Comment:
This is duplicated from the analyzer as I didn't see an obvious place to share code between the two.